test/extensions/footnote_block.unit

```diff
@@ -0,0 +1,80 @@
+>>> footnote with paragraph
```

Comment:
Also it looks like leading whitespace is allowed in the ref?

```markdown
Footnote 3 link[^ a].

[^a]: Text
```

works in GitHub.

Reply:
now supported